cooltools/expected.py

```diff
@@ -266,28 +266,41 @@ def count_bad_pixels_per_block(clr, supports, weight_name="weight", bad_bins=Non
         raise NotImplementedError("providing external list \
             of bad bins is not implemented.")
 
+    # Get the total number of bins per region
+    n_tot = [np.diff(clr.extent(region)) for region in regions]
```

Comment:
1. the typo here `regions` should be `supports` ...
2. `np.diff` returns `ndarray` we should extract a scalar from it `np.diff().item()` is the new way:
```
DeprecationWarning: np.asscalar(a) is deprecated since NumPy v1.16, use a.item() instead
```

Reply:
i PRed that to `nvictus/master`